packages/editor/src/components/global-keyboard-shortcuts/register-shortcuts.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * WordPress dependencies
+ */
+import { useEffect } from '@wordpress/element';
+import { useDispatch } from '@wordpress/data';
+import { __ } from '@wordpress/i18n';
+import { BlockEditorKeyboardShortcuts } from '@wordpress/block-editor';
+
+function EditorKeyboardShortcutsRegister() {
```

Comment:
Nit: There's inconsistency here in that the file is named as "register shortcuts" but the component as "shortcuts register".